### PR TITLE
Carthage support with updated README

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,7 +94,7 @@ github "hyperoslo/Tabby"
 
 ## Author
 
-Made by Hyper Interaktiv AS. Contact us at ios@hyper.no.
+Made by Hyper Oslo. Contact us at ios@hyper.no.
```
Comment (onmyway133) [inline README.md:97]: Oh so the framework template needs to be changed as well

Comment (RamonGilabert) [reply README.md:97]: Yeah that would be nice. Hyper Interaktiv AS is fine I guess. But I think it's old.

**Comment by onmyway133** [comment]
💥

File: README.md (hunk 1 above)
Comment (zenangst) [reply README.md:97]: It is just the legal name of the company. It could just say Hyper.

**Pull request merged**